Automattic/wp-calypso
# Redesign the cancel campaign progress dialog to match design

Related to #

## Proposed Changes

* Readapted the dialog when cancelling creating a campaign to match designs

## Testing Instructions
- Create a campaign, click on " < Back" to cancel it. A dialog should appear, the dialog should be similar to this:
![image](https://github.com/Automattic/wp-calypso/assets/43957544/08d968db-ad54-47a5-8276-3ba303156ca7)


## Pre-merge Checklist
